src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+    <Collapser
+        id="Device"
+        title="Device"
+    >
+Break out performance and engagement device type so you can understand:
+
+* Typical breakdown of desktop vs mobile browser users
+* Experience of desktop vs mobile browser users
+
+Facet by `deviceType`.
+
+Related attributes:
+* `userAgentName`
+* `userAgentOS`
+* `userAgentVersion`
+    </Collapser>
+    <Collapser
+        id="Product"
+        title="Product or line of business"
+    >
+When a product is a separate line of business or service provided by an organization. Some examples of industries and respective products:
+
+* An insurance company that sells both car and house insurance.
+* A media company that has multiple streaming services or channels.
+* A travel company that provides car rental as well as hotel bookings.
+
+**Basic:** Break out performance by product by:
+* Faceting on `pageUrl`: Use this approach when you group multiple products into one browser app in New Relic.
+* Faceting by `appName`: Use this approach when you instrument each product as a separate web app.
+* Grouping by `appName` and then facet: Use this approach when you hsbr multiple apps in browser supporting one product.
+
+**Advanced:** Add product offerings as custom attributes to browser pages using [custom attributes](/docs/telemetry-data-platform/custom-data/custom-events/report-browser-monitoring-custom-events-attributes).
+    </Collapser>
+    <Collapser
+        id="Environment"
+        title="Environment"
+    >
+A naming convention that specifies the environment for browser apps during or after instrumentation . Well named browser apps specify products and/or functions as well as environment. Examples:
+* `account-management.prod`
+* `hotels-book.prod`
+* `car-insurance.uat`
+
+Use app naming conventions to note that the environment supports filtering data in both the UI and in dashboards. For more information, see our docs for [how to rename browser apps](/docs/browser/new-relic-browser/configuration/rename-browser-apps/).
+    </Collapser>
+    <Collapser
+        id="Team"
+        title="Team"
+    >
+In some organizations, a single team supports multiple products, while in others, a product is big enough to be supported by multiple teams. Report on team performance against customer experience or engagement by either adding the team name to the browser app name in New Relic (for example, `account-management.prod.unicorn-squad`) or by using [custom attributes](/docs/telemetry-data-platform/custom-data/custom-events/report-browser-monitoring-custom-events-attributes).
+    </Collapser>
+</CollapserGroup>        
+    </Step>
+    <Step>
+### Import the quality foundation dashboard
+
+<img
+  title="Customer experience quality foundation dashboard"
+  alt="Customer experience quality foundation dashboard"
+  src={cxQfDashboard}
+/>
+
+The next step is to create a dashboard that you'll use to measure and improve your customer experience against your baselines. To do this: 
```

Comment:
"The next step is to create a dashboard you can use to measure and improve your customer experience against your baseline performance. To do this:"

Reply:
Thank you!